honssh/output_handler.py

```diff
@@ -233,10 +240,12 @@ def channel_opened(self, channel_id, channel_name):
         plugins.run_plugins_function(self.loaded_plugins, 'channel_opened', True, channel)
 
     def channel_closed(self, channel):
+        self._channel_closed(channel.uuid)
```

Comment:
What's this for? Why the _channel_closed function instead of just channel_closed?